tests/unit/amo/components/TestLandingPage.js

```diff
@@ -214,7 +214,6 @@ describe(__filename, () => {
     });
 
     expect(root).toIncludeText('Featured extensions');
-    expect(root).toIncludeText('See more featured extensions');
```

Comment:
ugh, this suite is not a good unit test of the component. One day we should rewrite the `renderAndMount()` parts.